src/css/custom.css

```diff
@@ -76,3 +76,80 @@ img + br + em::before {
     flex: 1;
     min-width: 20rem;
 }
+
+.typography h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+    margin: 0px 0px 0.613rem !important;
+    font-family: "Segoe UI", "Helvetica Neue", "Roboto", Helvetica, Arial,
+        sans-serif !important;
+}
+.typography h1 {
+    font-size: 2rem;
+    font-weight: 300;
+    line-height: 1.167;
+}
+.typography h2 {
+    font-size: 1.812rem;
+    font-weight: 300;
+    line-height: 1.167;
+}
+.typography h3 {
+    font-size: 1.625rem;
+    font-weight: 400;
+    line-height: 1.167;
+}
+.typography h4 {
+    font-size: 1.438rem;
+    font-weight: 500;
```

Comment:
One thing I've found is that the threshold between levels of boldness can vary between browsers. One good example from our own designers is "Segoe UI" around a weight of 500. In Chrome, 499 looks plain and 500 looks bold, but in Firefox 500 looks plain and 501 looks bold. In production this can be seen by comparing dashboard tiles between Firefox and Chrome. They are "Segoe UI" with a font weight of 500 so they look plain in Firefox but bold in Chrome.

I think that we should be advising use of font-weights that are not right on the edge like that. A value of 510 or 550 would be safer.

It does seem to vary based on the font, too. For Times New Roman, Chrome's threshold is 549-550 but Firefox's is 599-600.

Reply:
Thanks @kenlyon. Font weights are though to control across fonts, browsers, and operating systems. We will try to get as close as possible but I don't expect we will achieve parity. As long as the differences are noticeable in each browser/OS then we will be happy.

Reply:
![chrome-win10](https://github.com/geocortex/meridian-design/assets/5566971/030114d4-a774-4bd0-a432-9ee98b37574b)
![ff-win10](https://github.com/geocortex/meridian-design/assets/5566971/d45de9e0-7ba1-436e-9bc0-1b20ab300aaa)
![ff-macos-helvetica](https://github.com/geocortex/meridian-design/assets/5566971/9cb9845c-9a71-4962-8b17-9fc56a0fc35a)
![chrome-macos-helvetica](https://github.com/geocortex/meridian-design/assets/5566971/fb63a66b-6b13-4737-ae2e-410a82eaf2dc)